pipelines/disable-resource-locks.yaml

```diff
@@ -15,6 +33,6 @@ jobs:
       inputs:
         scriptType: bash
         scriptLocation: scriptPath
-        azureSubscription: '$(azureSubscription)'
+        azureSubscription: $(${{ parameters.subscription }})
```

Comment:
you have typo'd `$(${{ parameters.subscriptions }})`

Reply:
Updated